docs/configuration.md

```diff
@@ -38,9 +38,21 @@ These steps should be repeated for each project-version before using any zanata-
 
 You can customize `zanata.xml` with command hooks so that other tools will automatically run before or after Zanata commands. Read about command hooks at the [command hook page on the wiki](https://github.com/zanata/zanata-server/wiki/Client-Command-Hooks).
 
+## Source directory and translation directory
+
+`<src-dir>` and `<trans-dir>` can be defined in zanata.xml. You should use a relative path relative to the directory where zanata.xml lives. This will save you from giving the option on command line each time.
+
+Example:
+
+```xml
+<src-dir>po</src-dir>
+<trans-dir>.</trans-dir>
+```
 
 ## Locale Configuration
 
+***NOTE:*** Once feature [1156236](https://bugzilla.redhat.com/show_bug.cgi?id=1156236) is released, you don't need to define any locales in your zanata.xml anymore.
```

Comment:
Can we just say which server version it will be supported in? We can mention to look at the bottom of any page on the server for the version?

> **Note:** Locale configuration in zanata.xml is being phased out. When your project is hosted on a server of version X.X or above, you should remove this configuration. The server version is shown at the bottom of any page on the server.